client/ayon_core/plugins/publish/validate_version.py

```diff
@@ -1,6 +1,10 @@
 import pyblish.api
+
+from ayon_core.lib import filter_profiles
 from ayon_core.pipeline.publish import (
-    PublishValidationError, OptionalPyblishPluginMixin
+    PublishValidationError,
+    OptionalPyblishPluginMixin,
+    get_current_host_name,
 )
```

Comment:
This PR broken this validator. This import is wrong.
It is actually in `ayon_core.pipeline.context_tools`.